library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -100,7 +100,7 @@ def main
       @CardRegex =
         # other: irlan|lo|plip|...
         {
-          "netcard" => "ath|bnep|ci|ctc|slc|dummy|bond|escon|eth|fddi|ficon|hsi|qeth|lcs|iucv|myri|tr|usb|wlan|xp|vlan|br|tun|tap|ib|em|p|p[0-9]+p",
+          "netcard" => "ath|ci|ctc|slc|dummy|bond|eth|ficon|hsi|qeth|lcs|wlan|xp|vlan|br|tun|tap|ib|em|p|p[0-9]+p",
```

Comment:
I am not sure if we should do it before we have ready that unsupported card type. Also some other entries here looks a bit strange like ath, ci, xp, p and pXp.

Reply:
Yep, I was not sure about some of the because of not mentioned in the feaure request. 'ath' should be recognized as wlan if I'm not wrong. 'xp' removed, 'ppp' we already have a request for adding them back... will check all of them carefully